worlds/sm64ex/Options.py

```diff
@@ -73,6 +73,7 @@ class AreaRandomizer(Choice):
     option_Off = 0
     option_Courses_Only = 1
     option_Courses_and_Secrets = 2
+    option_Courses_and_Secrets_Separate = 3
```

Comment:
Swap Options 2 and 3 please, think that makes more sense (Something like "increasing difficulty")

Reply:
Done